### PR TITLE
🌱 Use deterministic Faker seed

### LINKED ISSUE #162: 'Share birth record' Percy screenshot has random data
Anything with random data creates false Percy diffs and slows down merging

Screen Shot 2019-08-29 at 10 38 37 AM

### PR DESCRIPTION
**Pull request by mermop**
Set the Faker seed so that we create names in the same order and stop confusing Percy - see [Faker docs](https://github.com/faker-ruby/faker#deterministic-random)

- Fixes #162 
- Removes the numbers from first and middle names
- Uses NZ locale for Faker

### CODE REVIEW

**Comment by mermop** [comment]
observe that these are the same - [percy build 1](https://percy.io/Lab/feijoa/builds/2480448?utm_campaign=Lab&utm_content=feijoa&utm_source=github_status_public), [percy build 2](https://percy.io/Lab/feijoa/builds/2480484?utm_campaign=Lab&utm_content=feijoa&utm_source=github_status_public)

**Review by Br3nda — COMMENTED** [review]
We can use `Faker::Name.middle_name`

**Review by Br3nda — COMMENTED** [review]
Is there a way to make the names less Anglo?

**Comment by mermop** [comment]
@Br3nda I have a faker PR to make NZ locale names reflect actual usage in NZ - changing it to the NZ locale should introduce a few Māori names but not that many right now - 
 this will introduce more https://github.com/faker-ruby/faker/pull/1568 - but I need to go back and write some tests for it..

File: spec/factories/birth_record.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 FactoryBot.define do
   factory :birth_record do
-    sequence(:first_and_middle_names) { |n| "#{Faker::Name.first_name} #{n}" }
+    first_and_middle_names { Faker::Name.name_with_middle }
```
Comment (Br3nda) [inline spec/factories/birth_record.rb]: this makes a 3 name long name. How about using Name.middle instead?

Comment (mermop) [reply spec/factories/birth_record.rb]: Done

**Review by Br3nda — APPROVED** [review]

**Pull request merged**